aws-ts-assume-role/README.md

```diff
@@ -0,0 +1,66 @@
+# AWS AssumeRole Example
+
+This example demonstrates how to use the AssumeRole functionality of the AWS provider in order to create resources in
+the security context of an IAM Role assumed by the IAM User running the Pulumi program.
+
+## Deploying the Example 
+
+These instructions assume you are familiar with running Pulumi programs written in TypeScript. Some other examples 
```

Comment:
hanging fragment?

Reply:
Yes, I’ll correct this before merging. Good catch.

Reply:
I've added some examples which walk through each step here.